examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>To use geodesic costs defined on a graph, we can create the underlying graph (here in gene expression space) using [...]. Note that the <code>cost</code> has to be set explicitly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>